test/util/util.go

```diff
@@ -59,12 +61,34 @@ func WaitForIssuerCondition(client clientset.IssuerInterface, name string, condi
 			return issuer.HasCondition(condition), nil
 		},
 	)
+	return addIssuerLastStatusConditionByType(client, pollErr, name, condition.Type)
+}
+
+// try to retrieve last condition to help diagnose tests.
+func addIssuerLastStatusConditionByType(client clientset.IssuerInterface, pollErr error, name string, conditionType v1alpha1.IssuerConditionType) error {
```

Comment:
maybe rename to `wrapErrorWithIssuerStatusCondition`? took me a little while to work out what this was doing